host_guest/cyclodextrin_derivatives/CD_submission.txt

```diff
@@ -0,0 +1,91 @@
+# Results for clip
+#
+# This file will be automatically parsed.  It must contain the following five elements: predictions, name of method, software listing, method, and ranked.
+# These elements must be provided in the order shown.
+# Please explicitly describe how you handle ions and pKa effects.
+#
+# The data in each prediction line should be structured as follows, with all (up to six) numbers in kcal/mol.
+# host-guest ID (note that the host varies!), Free energy, free energy SEM, free energy method error, enthalpy, enthalpy SEM, enthalpy method uncertainty
+# Note that the "method error" should be your estimate of ACCURACY of this particular approach
+# for the compound considered.
+# Beta CD (bCD) is optional and will not be judged, as literature values are available.
+# The list of predictions must begin with the "Prediction:" keyword, as illustrated here.
+Predictions:
+bCD-g1, -10.1,,,  -0.3,,
```

Comment:
Just curious, is there any particular reason the formatting is suppose to be multiple commas? readability?

Reply:
No, there's room for more fields there. (Commas delimit the fields).

Reply:
(I made the first row have all fields filled).